clients/src/test/java/org/apache/kafka/common/network/NetworkTestUtils.java

```diff
@@ -103,7 +103,12 @@ public static ChannelState waitForChannelClose(Selector selector, String node, C
             selector.poll(100L);
             if (selector.channel(node) == null && selector.closingChannel(node) == null) {
                 closed = true;
-                break;
+                ChannelState currentState = selector.disconnected().get(node);
```

Comment:
is it related to this change?

Reply:
it was related to the failing teest with was commented out elsewhere. removed now.